libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxSecondaryManeuver.kt

```diff
@@ -6,6 +6,7 @@ import androidx.appcompat.widget.AppCompatTextView
 import androidx.core.content.ContextCompat
 import com.mapbox.navigation.ui.maneuver.R
 import com.mapbox.navigation.ui.maneuver.model.ManeuverInstructionGenerator
+import com.mapbox.navigation.ui.maneuver.model.RoadShield
 import com.mapbox.navigation.ui.maneuver.model.SecondaryManeuver
 
```

Comment:
Noticed that we're still using `@JvmOverloads` here https://github.com/mapbox/mapbox-navigation-android/issues/4134